src/lib/viewers/image/MultiImageViewer.js

```diff
@@ -255,6 +260,10 @@ class MultiImageViewer extends ImageBaseViewer {
      */
     loadUI() {
         super.loadUI();
+
+        this.zoomControls = new ZoomControls(this.controls);
```

Comment:
Looks like this is already handled by the base class. Should the `init` call move to the base class, as well?